integration/indexes_compat_test.go

```diff
@@ -751,3 +751,124 @@ func TestDropIndexesCommandCompat(t *testing.T) {
 		})
 	}
 }
+
+func TestCreateIndexesUniqueCompat(t *testing.T) {
+	setup.SkipForTigrisWithReason(t, "Indexes creation is not supported for Tigris")
+
+	t.Parallel()
+
+	for name, tc := range map[string]struct { //nolint:vet // for readability
+		models    []mongo.IndexModel // required, index to create
+		insertDoc bson.D             // required, document to insert for uniqueness check
+		new       bool               // optional, insert new document before check uniqueness
+
+		skip string // optional, skip test with a specified reason
+	}{
+		"ExistingFieldIndex": {
+			models: []mongo.IndexModel{
+				{
+					Keys:    bson.D{{"v", 1}},
+					Options: new(options.IndexOptions).SetUnique(true),
```

Comment:
Could you add a test for non boolean value of  unique please? I'm not sure if it returns error or succeeds, you may be able to add a test case to this in case of error https://github.com/FerretDB/FerretDB/blob/main/integration/indexes_test.go#L176

Reply:
Added [here](https://github.com/FerretDB/FerretDB/pull/2814/files#diff-b2e4c964b500457dfe583f5f7f7138a8871544e5338b53bbc0d8a02721a9e46bR375).

Reply:
Added a test to `TestCreateIndexesInvalidSpec`.

Reply:
Perfect, thanks!